packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion-group/button/default.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Returns the default styles for component `suggestion group button`.
+ *
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function suggestionGroupButtonDefault() {
+  return {
+    display: 'grid',
+    gridAutoFlow: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignContent: 'center',
+    boxSizing: 'border-box',
+    backgroundColor: 'transparent',
+    cursor: 'pointer',
+    color: 'currentColor',
+    minHeight: 'inherit',
+    aspectRatio: '1',
+    '&:hover': {
+      transform: 'scale(1.2)'
```

Comment:
Just in case: in XDS Figma the scale default value is set to `1.4`.

Reply:
We decided to reduce the default scale and it seems that the figma doc was not updated. It is correct the 1.2.